policyengine_us/parameters/calibration/gov/cbo/income_by_source.yaml

```diff
@@ -148,6 +510,76 @@ taxable_pension_income:
   2033-01-01: 2_373_800_000_000
   2034-01-01: 2_467_700_000_000
   2035-01-01: 2_571_400_000_000
+  # Extended projections (2036-2100) using SSA GDP growth
+  # Methodology: Applied SSA GDP growth rates to maintain consistency with AGI
+  # Pension income maintains 9.85% of AGI (2035 ratio)
+  # Reflects wage growth and demographic trends (aging population)
+  2036-01-01: 2_628_042_715_481
+  2037-01-01: 2_731_995_715_176
+  2038-01-01: 2_840_661_250_857
+  2039-01-01: 2_953_522_736_754
+  2040-01-01: 3_069_413_291_785
+  2041-01-01: 3_189_598_009_043
+  2042-01-01: 3_314_531_943_658
+  2043-01-01: 3_444_397_486_077
+  2044-01-01: 3_579_741_155_817
+  2045-01-01: 3_720_538_177_695
+  2046-01-01: 3_867_290_997_039
+  2047-01-01: 4_020_651_063_611
+  2048-01-01: 4_180_531_990_388
+  2049-01-01: 4_347_532_146_419
+  2050-01-01: 4_521_726_511_549
+  2051-01-01: 4_703_488_677_564
+  2052-01-01: 4_893_299_145_925
+  2053-01-01: 5_091_658_959_098
+  2054-01-01: 5_298_805_421_949
+  2055-01-01: 5_515_294_468_697
+  # Long-term projections (2056-2100)
+  2056-01-01: 5_741_737_093_468
+  2057-01-01: 5_978_862_730_656
+  2058-01-01: 6_227_400_846_094
+  2059-01-01: 6_488_188_044_558
+  2060-01-01: 6_762_105_018_527
+  2061-01-01: 7_050_054_009_892
+  2062-01-01: 7_352_965_632_690
+  2063-01-01: 7_671_709_006_695
+  2064-01-01: 8_007_222_699_513
+  2065-01-01: 8_360_434_166_663
+  2066-01-01: 8_732_401_802_928
+  2067-01-01: 9_124_243_896_803
+  2068-01-01: 9_537_092_516_619
+  2069-01-01: 9_972_212_024_754
+  2070-01-01: 10_430_793_691_176
+  2071-01-01: 10_914_127_479_782
+  2072-01-01: 11_423_671_479_652
+  2073-01-01: 11_961_142_597_088
+  2074-01-01: 12_528_472_896_655
+  2075-01-01: 13_127_797_055_929
+  2076-01-01: 13_761_525_169_419
+  2077-01-01: 14_431_362_639_032
+  2078-01-01: 15_139_399_491_997
+  2079-01-01: 15_888_137_933_424
+  2080-01-01: 16_679_414_032_943
+  2081-01-01: 17_515_324_013_929
+  2082-01-01: 18_398_284_408_686
+  2083-01-01: 19_331_044_669_055
+  2084-01-01: 20_316_612_646_498
+  2085-01-01: 21_358_233_768_916
+  2086-01-01: 22_459_454_857_768
+  2087-01-01: 23_624_193_074_655
+  2088-01-01: 24_856_685_878_832
+  2089-01-01: 26_161_648_149_172
+  2090-01-01: 27_543_199_186_656
+  2091-01-01: 29_005_915_799_169
+  2092-01-01: 30_554_851_695_089
+  2093-01-01: 32_195_555_453_012
+  2094-01-01: 33_933_999_699_597
+  2095-01-01: 35_776_638_914_688
+  2096-01-01: 37_730_467_821_819
+  2097-01-01: 39_802_997_607_421
+  2098-01-01: 42_001_394_999_819
+  2099-01-01: 44_333_456_844_838
+  2100-01-01: 46_807_688_994_397
 taxable_social_security:
```

Comment:
are any other sources used as upratings for inputs?

Reply:
I think we've got them all